src/frontends/tensorflow/src/decoder_proto.cpp

```diff
@@ -263,10 +263,7 @@ ov::Any DecoderProto::get_attribute(const std::string& name) const {
                                 name,
                                 "' attribute is not supported.");
     case ::tensorflow::AttrValue::ValueCase::kFunc:
-        FRONT_END_GENERAL_CHECK(false,
-                                "Conversion from Tensorflow to OpenVINO data type failed: Function type for '",
-                                name,
-                                "' attribute is not supported.");
+        return attrs[0].func().name();
```

Comment:
Please leave a comment describing that we will get function using a different method, but we need the name of the function.

Reply:
Done.